Cargo.toml

```diff
@@ -24,6 +24,7 @@ parking_lot = "0.12"
 prometheus-client-derive-encode = { version = "0.4.1", path = "derive-encode" }
 prost = { version = "0.12.0", optional = true }
 prost-types = { version = "0.12.0", optional = true }
+log = "0.4.22"
```

Comment:
I try to keep the dependency footprint of this crate as small as possible. Adding `log` just for this use-case is not an option. Mind reverting it?